src/api/artifact/model.go

```diff
@@ -88,7 +88,8 @@ func (a *Artifact) ToSwagger() *models.Artifact {
 // Tag is the overall view of tag
 type Tag struct {
 	tag.Tag
-	// TODO add other attrs: signature, label, immutable status, etc
+	Immutable bool
```

Comment:
Add the property in swagger and handle it in the upper function `func (a *Artifact) ToSwagger() *models.Artifact`

Reply:
done.